.gitignore

```diff
@@ -1,4 +1,11 @@
 node_modules
 .nyc_output
 .vscode
-coverage
\ No newline at end of file
+coverage
+
+.DS_Store
```

Comment:
I believe this line is enough to ignore this file anywhere. No need to add the rest.

Additionally, I recommend adding those generic additions to your own global .gitignore file in`~/.gitignore`.